Makefile

```diff
@@ -6,6 +6,3 @@ check-linting: ## Checks the linting for code
 
 fix-linting: ## Fixes the linting in code
```

Comment:
Imo instead of a Makefile these should be in the `scripts` section of composer.json (like they are in opentracing-javascript).
I assume that PHP developers are more familiar with `composer run-script` than with `make`, and `make` is cumbersome to install on Windows.